### DISCUSSION TITLE
#35合肥电信2 https隧道建立之后http重定向不生效

### DISCUSSION THREAD

**Discussion by Mr-XiaoYi** [natfrp/wiki#388] [使用问题] [answered]
域名alist.scczz.com   {这是测试站，随时下线}

https访问正常，http访问显示503报错，后台已经部署301重定向，当然302我也部署过，奇怪的是全都不生效
想能不能让管理员看到修复一下，当然免费节点我很开心QAQ

**Comment by fakeboboliu** [comment, marked as answer]
此功能需要使用 sakurafrp 版本的客户端，
为了安全考虑，我们的大部分功能都在客户端本地实现，因此请尽可能使用我们分发的最新客户端。

https://doc.natfrp.com/frpc/manual.html
